Description: fix compile flags in Makefile
Author: Kevin Coyner <kcoyner@debian.org>,
        Marcos Talau <talau@users.sourceforge.net>
Bug-Debian: https://bugs.debian.org/928116
Last-Update: 2019-07-19
---
Index: mailcheck-1.91.2/Makefile
===================================================================
--- mailcheck-1.91.2.orig/Makefile
+++ mailcheck-1.91.2/Makefile
@@ -4,7 +4,7 @@ debug: mailcheck.c netrc.c netrc.h socke
 	$(CC) -Wall -O0 mailcheck.c netrc.c socket.c -g -o mailcheck
 
 mailcheck: mailcheck.c netrc.c netrc.h socket.c
-	$(CC) -Wall -O2 mailcheck.c netrc.c socket.c -s -o mailcheck
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -Wall mailcheck.c netrc.c socket.c -o mailcheck
 
 install: mailcheck
 	install mailcheck $(prefix)/usr/bin
